.ci/dockerfiles/Dockerfile.vllm-base

```
@@ -0,0 +1,44 @@
+ENV HF_HOME=${MODEL_DIR}/hf-home
+
+# iproute2 provides `ss`, used by the sanity's get_next_tcp_port helper to skip occupied
+# host ports (concurrent jobs share a SLURM node under host networking).
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends iproute2 \
+ && rm -rf /var/lib/apt/lists/*
+
+# huggingface_hub ships in the vLLM image; download once at build time so PR runs
+# never touch HuggingFace. A token is only needed for gated models (default is ungated).
+# Use the `hf` CLI (huggingface-cli is removed in recent huggingface_hub).
+RUN python3 -m pip install --no-cache-dir --upgrade "huggingface_hub[cli]" \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)